src/bin/cql-stress-cassandra-stress/settings/option/column.rs

```diff
@@ -57,9 +61,41 @@ impl ColumnOption {
     }
 }
 
+/// A type for parsing `-col n=` parameter.
+///
+/// In cassandra-stress, CLI originally accepts a distribution.
+/// However, only FIXED(?) distribution is supported by the tool.
```

Comment:
It is unclear to me whether `the tool` refers to cassandra-stress or cql-stress.